gems/pending/metadata/linux/LinuxSystemd.rb

```diff
@@ -32,7 +32,8 @@ def parse_systemd
     end
 
     def files(unit_extension)
-      (SYSTEM_DIRS + USER_DIRS).flat_map do |dir|
+      dirs = my_dirs
+      dirs.flat_map do |dir|
```

Comment:
I came across this SystemD issue when parsing a CentOS VM for this PR.  But you are correct, these should probably be added under a separate PR.  I can do so if you like.